app/views/ppm_estimator/index.html.erb

```diff
@@ -0,0 +1,156 @@
+    government will incentivize you to move yourself, you get to pocket the
+    difference!
+  </p>
+
+  <%= form_with url: ppm_estimator_path, class: 'ppm-form', id: 'ppm-estimate-form', method: 'get' do |form| %>
+    <fieldset>
+      <legend><div class="ppm-outline"></div>What is your rank?</legend>
+        <%= form.label :rank, 'Rank' %>
+        <%= form.select :rank, options_from_collection_for_select(@entitlements, :slug, :rank), { include_blank: true }, { id: :rank, required: true } %>
+    </fieldset>
+
+    <fieldset>
+      <legend><div class="ppm-outline"></div>What is your branch of service?</legend>
+        <%= form.label :branch, 'Branch of Service' %>
+        <%= form.select :branch, options_from_collection_for_select(@branches, :slug, :name), { include_blank: true }, { id: :branch, required: true } %>
+    </fieldset>
+
+    <fieldset>
+      <legend><div class="ppm-outline"></div>Do you have dependents?</legend>
+      <%= form.radio_button :dependents, 'yes', { id: :dependents_yes, required: true } %>
+      <%= form.label :dependents, raw('<strong>Yes</strong>, I have dependents'), :value => 'yes' %>
+      <%= form.radio_button :dependents, 'no', { id: :dependents_no, required: true } %>
+      <%= form.label :dependents, raw('<strong>No</strong>, I do not have dependents'), :value => 'no' %>
+    </fieldset>
+
+    <fieldset>
+      <legend><div class="ppm-outline"></div>Are you married?</legend>
+      <%= form.radio_button :married, 'yes', { id: :married_yes, required: true } %>
+      <%= form.label :married, raw('<strong>Yes</strong>, I am married'), :value => 'yes' %>
+      <%= form.radio_button :married, 'no', { id: :married_no, required: true } %>
+      <%= form.label :married, raw('<strong>No</strong>, I am not married'), :value => 'no' %>
+    </fieldset>
+
+    <fieldset>
+      <legend><div class="ppm-outline"></div>Where are you moving from and to?</legend>
+      <p>
+        To get the most accurate estimate, enter the locations authorized by
+        your orders. These locations might be different from where you live.
+      </p>
+      <div class="usa-grid-full grid-form-width">
+        <div class="usa-width-one-half">
+          <%= form.label :start, 'Origin ZIP Code' %>
+          <%= form.number_field :start, { id: :start, minlength: 5, maxlength: 5, inputmode: 'numeric', required: true } %>
+        </div>
+        <div class="usa-width-one-half">
+          <%= form.label :end, 'Destination ZIP Code' %>
+          <%= form.number_field :end, { id: :end, minlength: 5, maxlength: 5, inputmode: 'numeric', required: true } %>
+        </div>
+      </div>
+    </fieldset>
+
+    <fieldset>
+      <legend><div class="ppm-outline"></div>When do you want to move?</legend>
+      <%= form.label :date, 'Moving Date' %>
+      <%= form.date_field :date, { id: :date, pattern: '\d\d\d\d-\d\d?-\d\d?', placeholder: 'yyyy-mm-dd', required: true } %>
```

Comment:
I think you can simplify this `pattern` attribute to `\d{4}-\d{1,2}-\d{1,2}`, presuming you're expecting things like `2017-1-1` and `2017-01-01`.